runtime/near-evm-runner/tests/standard_ops.rs

```diff
@@ -337,7 +337,7 @@ fn test_meta_call_sig_and_recover() {
         // RLP encode of ["0x09", ["0x436170734C6F636B", "0x0123456789012345678901234567890123456789"]]
         hex::decode("e009de88436170734c6f636b940123456789012345678901234567890123456789").unwrap(),
     );
-    assert_eq!(hex::encode(&meta_tx3[0..65]), "0a2af43c3efab7ce535a00125b2505823c3c3218bacab1546a3e569ec15ca4557352f16ebabeeaa066a239346d7870afd49bf6e0b7b5c0d398d5cf894f3bdc8f1c");
+    assert_eq!(hex::encode(&meta_tx3[0..65]), "f4852afcf9b11e0c6bac70f0c468ed29fa5b2f1d5a786fc9b383729f591d4dd32a4e5bdec0e12881b9f469e14fb7eed1911613a8ea636f14df81e39568324f791b");
```

Comment:
Here i was made a mistake, method sig should only contains the method, not following custom types. corrected here and https://gist.github.com/ailisp/0f65bc1a3827b03141ae6601918a67a2/revisions and retest signature still matches with eth-sig-util produced